stl/inc/span

```diff
@@ -226,28 +226,30 @@ struct pointer_traits<_Span_iterator<_Ty>> {
 };
 
 // STRUCT TEMPLATE _Span_extent_type
-template <size_t _Extent>
+template <class _Ty, size_t _Extent>
 struct _Span_extent_type {
+    using pointer = _Ty*;
+
     constexpr _Span_extent_type() noexcept = default;

```

Comment:
Technically this should only be default constructible when `_Extent == 0` 

However, as we already prohibit other default constructors inside of span and we would need one implementation for concepts and one with classic SFINAE I would prefer to simply leave this as is

Reply:
Agreed - I see no need to replicate span's interface constraints in internal machinery.